### ISSUE TITLE
Internal Server Error - AZ Dems Nomad

### ISSUE THREAD

**Issue by flynnwastaken** [RagtagOpen/nomad#734]
On Chrome in Chromebook, clicking between homepage and Profile while logged into Admin account, both when selecting the homepage by clicking the logo in the upper left corner and when going to the Profile from the homepage using the dropdown user menu I've gotten "Internal Server Error"
![screenshot 2018-10-15 at 4 07 08 pm](https://user-images.githubusercontent.com/32378726/46983276-92b6f080-d094-11e8-9e67-88cd7e75a5d2.png)

**Comment by iandees** [comment]
This looks to be a transient error here: https://sentry.ragtag.org/ragtag/nomad-prod/issues/157/

**Comment by jillh510** [comment]
@iandees this bug is fixed, right?

**Comment by iandees** [comment]
We didn't do anything to fix it, but I don't think it's reproducible.

**Comment by kcaffall** [comment]
I have not repro'd this, but wondering if it could be caused by bouncing around environments (something Flynn would do, but clients wouldn't)? Different tabs being in production, AZ Dems, NY, like that.

**Comment by dryan** [comment]
Yeah I agree with Ian that this is transient. I did come across https://stackoverflow.com/questions/31121948/flask-sqlalchemy-cant-reconnect-until-invalid-transaction-is-rolled-back looking up the error message. I wonder if it was a bad migration?

**Comment by iandees** [comment]
My guess is that it was an error that happened in another session on this Nomad instance. I poked through the logs quickly to see if there was something that stood out before this error but didn't see anything.

**Comment by dryan** [comment]
Yeah I agree. I think we're good to close this out (leave it to @jillh510 to decide for sure). If we see this error in production I would suggest re-deploying on Heroku to get a new session opened up.

**Comment by jillh510** [comment]
I've taken it off the Priority list but would like to leave it open in case anyone sees it again in the next two weeks. If not, then we'll close as "can't reproduce".
